kmir/src/kmir/testing/fixtures.py

```diff
@@ -13,6 +13,10 @@
 
     from pytest import FixtureRequest, Parser
 
+import sys
+
+def pytest_configure(config) -> None:
+    sys.setrecursionlimit(1000000)
```

Comment:
My test program is too big (for the equation generation) so this is necessary , even after splitting it in two.
The `kmir` app already had it for a long time, now the tests have a larger stack as well.